wlc/__init__.py

```diff
@@ -326,15 +327,15 @@ def create_project(
 
         return self.post("projects/", **data)
 
-    def create_component(self, project, **kwargs):
+    def create_component(self, project, files=None, **kwargs):
         """Create a new component for project in the instance."""
 
         required_keys = ["name", "slug", "file_format", "filemask", "repo"]
         for key in required_keys:
             if key not in kwargs:
                 raise WeblateException(f"{key} is required.")
 
-        return self.post(f"projects/{project}/components/", **kwargs)
+        return self.post(f"projects/{project}/components/", files, **kwargs)
```

Comment:
Make it clear that this is files.

```suggestion
        return self.post(f"projects/{project}/components/", files=files, **kwargs)
```